cocotb/regression.py

```diff
@@ -180,16 +185,14 @@ def _discover_tests() -> Iterable[Test]:
                 _logger.info(traceback.format_exc())
                 raise
 
-            if test_str:
-
+            if tests is not None:
```

Comment:
```suggestion
            if tests is not None:
                not_found_tests = []
```

Reply:
Changed in 0d656990c6bc950012502e9d25ff767c6ea59468.